### PR TITLE
Added unit testing to the project

### PR DESCRIPTION
**Pull request by AndresConfortiGL**
## Description
6th PR - Added unit testing to the project.

## Story Type
- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?
It can be tested by running the coverage tests.

## Evidences
| Coverage Test  |
| ------------- |
|coverage|

## Checklist

- All unit Test passed :white_check_mark:

## Tested on
The project was tested on Android 8.0 / API 26 using the following devices:

- Nexus S
- Pixel 3a

## Package Structure
| Structure |
| ------------- |
|package|

### CODE REVIEW

File: app/src/test/java/com/mobile/tandil/javabaseproject/ReserverPresenterUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        presenter.showCheckOutDateFragment(listener);
+        Calendar dateCheckOut = Calendar.getInstance();
+        Calendar timeCheckOut = Calendar.getInstance();
+        presenter.saveDateListener(dateCheckOut, listener);
+        presenter.saveTimeListener(timeCheckOut);
+
+        presenter.saveParkingData(VALID_STRING_PARKING_LOT, VALID_STRING_SECURITY_CODE);
+
+        Mockito.verify(view).displayInvalidDateToast();
+    }
```
Comment (jnicolaslaurenti) [inline app/src/test/java/com/mobile/tandil/javabaseproject/ReserverPresenterUnitTest.java:120]: Can we meet to see this?

**Review by jnicolaslaurenti — APPROVED** [review]

File: app/build.gradle [hunk 2]
```diff
@@ -36,7 +36,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+    implementation 'com.google.ar.sceneform:filament-android:1.17.1'
```
Comment (DiegoDiazGlobant) [inline app/build.gradle]: Why do you add this import? Remove it please

**Review by kearneyeduardo — APPROVED** [review]

**Review by SartiMau — COMMENTED** [review]
+1 after comments solved

File: app/build.gradle (hunk 2 above)
Comment (AndresConfortiGL) [reply app/build.gradle]: Fixed

**Review by DiegoDiazGlobant — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**